api/payload.go

```diff
@@ -241,11 +246,17 @@ type CreateGamePayload struct {
 //Validate the create game payload
 func (p *CreateGamePayload) Validate() []string {
 	v := NewValidation()
-	sortedLevels := util.SortLevels(p.MembershipLevels)
-	minMembershipLevel := sortedLevels[0].Value
 
-	v.validateRequiredString("name", p.Name)
+	var minMembershipLevel int
+
 	v.validateRequiredMap("membershipLevels", p.MembershipLevels)
+
+	if p.MembershipLevels != nil {
+		sortedLevels := util.SortLevels(p.MembershipLevels)
+		minMembershipLevel = sortedLevels[0].Value
+	}
```

Comment:
This will panic when `MembershipLevels` is an empty object.

`len(map)` returns 0 when the input is `nil`. So we can use it.

```suggestion
	if len(p.MembershipLevels) >= 0 {
		sortedLevels := util.SortLevels(p.MembershipLevels)
		minMembershipLevel = sortedLevels[0].Value
	}
```